### DISCUSSION TITLE
Ubuntu 24.04 not booting after installing fuse

### DISCUSSION THREAD

**Discussion by bkanwasi** [AppImage/AppImageKit#1330] [Q&A]
I was trying to run an AppImage when it asked me to install fuse. I did so, then rebooted, and I encountered these error messages: '[drm] psp gfx command LOAD TA(0x1) failed and response status is (0x7)' and '[drm] psp gfx command INVOKE CMD (0x3) failed and response status is (0x4)'. Additionally, I received these messages: 'amdgpu 0000:03:00.0: amdgpu: Secure display: Generic Failure' and 'amdgpu 0000:03:00.0: amdgpu: SECUREDISPLAY: query securedisplay TA failed. ret 0xe'.

After that, I went into recovery mode and managed to access gd3. However, after entering my password, the screen went white. I can still see my cursor, but nothing else. Can anyone help me?

blob:https://web.whatsapp.com/74cf1081-796d-462b-aeb4-fb984bb8de96

![74cf1081-796d-462b-aeb4-fb984bb8de96](https://github.com/AppImage/AppImageKit/assets/18642598/36295b4b-d7a8-4802-855a-7b6ebce13771)
![f83d0779-34b1-4bf2-8a32-bf5d08f8291d](https://github.com/AppImage/AppImageKit/assets/18642598/9fcf1c72-37d3-4a34-8b3c-f03ae1ebae3c)

Could you please provide instructions for installing FUSE on Ubuntu 24.04?

Thanks

**Comment by probonopd** [comment]
Enable universe, then `sudo apt-get -y install libfuse2`
